go/sign/sign.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"crypto/hmac"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/aws/aws-lambda-go/lambda"
+)
+
+type event struct {
+	URL    string                 `json:"url"`
+	Secret string                 `json:"secret"`
+	Body   map[string]interface{} `json:"body"`
+}
+
+func signBody(secret, body []byte) string {
+	computed := hmac.New(sha256.New, secret)
+	computed.Write(body)
+	return "sha256=" + hex.EncodeToString(computed.Sum(nil))
+}
+
+func sendWebhookRequest(url string, secret []byte, body []byte) error {
```

Comment:
we should have this function return the response body from the downstream http endpoint (i'm guessing there's logs that would make that useful to have)